ST_stats.c

```diff
@@ -26,97 +26,23 @@
 #include "myMemory.h"
 #include "sw_src/SW_Site.h"
 #include "ST_structs.h"
+#include "ST_stats.h" // Contains most of the function declarations.
+#include "ST_initialization.h"
```

Comment:
Unused include. "stdlib.h" and "sw_src/SW_Site.h" are also unused.

Reply:
Resolved by 5b1b919.